Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM alpine:3.4
+FROM plugins/base:multiarch
+MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
 
-RUN apk update && \
-  apk add \
-    ca-certificates && \
-  rm -rf /var/cache/apk/*
+LABEL org.label-schema.version=latest
```

Comment:
I've been thinking about this, does this make sense/what does it mean when the version says it is latest?

Reply:
These labels will be shown on pages like https://microbadger.com/images/plugins/slac. I thought about displaying the real version there, but than it's a third place that needs to be updated which will be forgotten for sure